package.json

```diff
@@ -2,17 +2,23 @@
   "name": "awesome-imba",
   "version": "1.0.0",
   "description": "A curated list of awesome Imba frameworks, libraries, software and resources",
+  "engines": {
+    "node": ">=8"
```

Comment:
Also not sure that we need >=8 if we migrate `scripts/check-branch-name.js` to the `Imba`